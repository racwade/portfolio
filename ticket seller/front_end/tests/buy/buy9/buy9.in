login
test_buy
buy
test_sell
Few
2
logout