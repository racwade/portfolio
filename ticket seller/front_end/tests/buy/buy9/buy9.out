welcome to the ticket seller system
enter username:
logged in as standard_buy
enter seller's username:
enter event name:
enter number of tickets to buy:
error: not enough tickets remaining
bye